### DISCUSSION TITLE
Urgent: GitHub Account Vanished 😢

### DISCUSSION THREAD

**Discussion by devkkaran** [community/community#137246] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear GitHub Support Team,

I am writing to seek urgent assistance with recovering my GitHub account. My account appears to have vanished, and I am unable to locate my account, repositories, or any of my activity on GitHub. I was previously able to log in using the GitHub mobile app, but now I cannot even access my account through the app. 

I attempted to reset my password using the "Forgot Password" feature, but I did not receive any reset password link in either of my associated email addresses. I am concerned that my GitHub account may have been hacked. Below are the details related to my account:

- **GitHub Username:** devkarank
- **Primary Email ID:** devkarankvaghasiya@gmail.com
- **Secondary Email ID:** vaghasiyadev47@gmail.com
- **Last Successful Login Date:** 29/08/2024
- **Repositories of Concern:** "Leetcode-solutions"
- **Two-Factor Authentication (2FA):** Enabled (I also have the recovery codes)
- **Recent Activity:** I recently generated a fine-grained personal access token for the LeetSync Chrome extension to sync my LeetCode problems to my GitHub repository. The issue began approximately two days after I granted access to this extension. The extension can be found here: [LeetSync GitHub Repository](https://github.com/LeetSync/LeetSync).

I have already raised a support ticket with the following details:

- **Ticket ID:** 2974706

I have tried all possible means to recover my account, but I am still unable to access it. Please advise on the next steps I should take to regain access to my account.

**Comment by elnurumo** [comment]
Same problem,

I hope this message finds you well. I am writing to seek assistance regarding the sudden suspension of my brother's GitHub account. His username is huseynovelmir, and the email associated with the account is huseynovelmir789@gmail.com.

To his knowledge, he has not violated any of GitHub's community guidelines or terms of service. We are unsure why his account was suspended, and we would greatly appreciate any clarification on the matter.

Could you please investigate the issue and let us know if there are any steps he can take to resolve it? He relies on GitHub for his projects and would like to restore access as soon as possible.

Thank you for your time and assistance.

Best regards,
Elnur

**Reply by devkkaran** [reply]
[elnurumo](https://github.com/elnurumo) My GitHub account has completely disappeared, and I cannot find it anywhere. I previously raised a support ticket regarding this issue, but the response I received was disappointing. Here’s what they said in mail:

---

**IMPORTANT: Support Ticket Declined**

Hi there,

We now require that new support requests be created using our Support website:

[GitHub Support Website](https://support.github.com)

Please use this website to search through resources that may help you find the solutions you are looking for or connect with our GitHub experts.

Thank you,  
GitHub Support

**Please do not reply to this email, as it will not be seen by our team.**

---

Has anyone else experienced something similar? If possible, could you try generating a ticket and let me know what response you get from GitHub support? [elnurumo](https://github.com/elnurumo), could you try generating a ticket and see what response you get from GitHub support? Please let me know about this.


---

**Comment by yashnaresh** [comment]
same issue ,support site given above is also not opening for me

**Comment by ebndev** [comment]
Hi @Devkkaran,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
